cspell.json

```diff
@@ -34,7 +34,9 @@
         "Cook'n",
         "Copernicium",
         "Coughlin",
+        "Cpolygon",
```

Comment:
should take this out as soon as #1656 is merged

Reply:
Merged in new file and it is gone.